data/announcements/scheduled.yaml

```diff
@@ -130,3 +130,13 @@ announcements:
   message: |
     5 days of incredible opportunities to collaborate, learn + share with the entire community!<br />
     October 24 - 28, 2022.
+
+- name: Freezing k8s.gcr.io
+  startTime: 2023-02-21T00:00:00 # Added in https://github.com/kubernetes/website/pull/39575
+                                 # This should run before and after Kubecon EU 2023
+  endTime: 2023-04-15T00:00:00
```

Comment:
Watch out for grammar. Ideally, we use a different wording from the 3rd of April.

Reply:
eg: “Legacy k8s.gcr.io container image registry was frozen on the 3rd of April 2023”